packages/lesmis-server/src/connections/EntityConnection.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+import camelcaseKeys from 'camelcase-keys';
+import { ApiConnection } from '@tupaia/server-boilerplate';
+import { LESMIS_PROJECT_NAME, LESMIS_COUNTRY_ENTITY_CODE } from '../constants';
+
+const { ENTITY_SERVER_API_URL = 'http://localhost:8050/v1' } = process.env;
+
+export class EntityConnection extends ApiConnection {
+  baseUrl = ENTITY_SERVER_API_URL;
+
+  async getEntities() {
```

Comment:
We are just getting all the data at the moment rather than specifying fields. This may need to change later depending on how the front end caching works out. Will add support for fields on an as needed basis.